<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlLearn</class>
 <widget class="QDialog" name="ControlLearn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="MidiRadio">
     <property name="text">
      <string>Midi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="OscRadio">
     <property name="text">
      <string>Osc</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="KeyboardRadio">
     <property name="text">
      <string>Keybord</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="ControlView"/>
   </item>
   <item>
    <widget class="QTableView" name="ControlList"/>
   </item>
   <item>
    <widget class="QCheckBox" name="AutoAcceptBox">
     <property name="text">
      <string>Auto Accept First Control</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ControlLearn</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ControlLearn</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MidiRadio</sender>
   <signal>clicked()</signal>
   <receiver>ControlList</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OscRadio</sender>
   <signal>clicked()</signal>
   <receiver>ControlList</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>KeyboardRadio</sender>
   <signal>clicked()</signal>
   <receiver>ControlList</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MidiRadio</sender>
   <signal>clicked()</signal>
   <receiver>ControlView</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OscRadio</sender>
   <signal>clicked()</signal>
   <receiver>ControlView</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>KeyboardRadio</sender>
   <signal>clicked()</signal>
   <receiver>ControlView</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
